string-calculator.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="assets/styles/string-calculator.css">
+    <title>Title</title>
+</head>
+<body>
+<main class="calculator">
+    <section id="window" class="calc-window"></section>
```

Comment:
_UI improvements_: there is no possibility to clear the input after the calculation ("C" button on a calculator)

Reply:
fixed